.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: linting
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: macos-latest
```

Comment:
You can use `ubuntu-latest` here, as the Ubuntu Actions runners now ship with Homebrew on Linux.